test/e2e/common_suite_test.go

```diff
@@ -48,15 +61,26 @@ func (tc *testCase) withSecret(secret *v1.Secret) *testCase {
 	tc.secret = secret
 	return tc
 }
-
```

Comment:
Can you re-add the blank lines between the functions please as I think it makes it easier to read with them in